integ-test/src/test/java/org/opensearch/sql/sql/CsvFormatIT.java

```diff
@@ -40,12 +40,12 @@ public void escapeSanitizeTest() {
         String.format(Locale.ROOT, "SELECT firstname, lastname FROM %s", TEST_INDEX_BANK_CSV_SANITIZE),
         "csv&sanitize=false");
     assertEquals(
-        "firstname,lastname\n"
-            + "+Amber JOHnny,Duke Willmington+\n"
-            + "-Hattie,Bond-\n"
-            + "=Nanette,Bates=\n"
-            + "@Dale,Adams@\n"
-            + "\",Elinor\",\"Ratliff,,,\"\n",
+        "firstname,lastname" + System.lineSeparator()
+            + "+Amber JOHnny,Duke Willmington+" + System.lineSeparator()
```

Comment:
You can use `%n` and `String.format`, this may be simpler. See example in `AggregationQueryBuilderTest`.

Reply:
Any advantage of using one over the other. I thought using System.lineSeparator would encapsulate for all os types.

Reply:
No advantage, `%n` does the same. It could be a bit more readable.

Reply:
i would also prefer `%n` for consistency with existing tests
https://github.com/opensearch-project/sql/blob/79f765c4a49ebcd441885a8d74e3353c757d4d8d/opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/aggregation/dsl/MetricAggregationBuilderTest.java#L66-L73